docs/coordinator/api/remote.md

```diff
@@ -30,6 +30,33 @@ docs/common/headers_optional_read_write.md
 
 Binary [snappy compressed](http://google.github.io/snappy/) Prometheus [WriteRequest protobuf message](https://github.com/prometheus/prometheus/blob/10444e8b1dc69ffcddab93f09ba8dfa6a4a2fddb/prompb/remote.proto#L22-L24).
 
+### Available Tuning Params 
+
+**Note:** All relevant parameters can be found under the `queue_config` section of the Prometheus `remote_write` configuration.
```

Comment:
I would just link to the Prometheus website so that this doesn't go out of date?

Reply:
Like just link, or add the link to what I already have?

Reply:
I would just have the link otherwise what you have might diverge at some point.

Reply:
Link added.